client/js/components/user/DpUserList/DpUserFormFields.vue

```diff
@@ -313,7 +314,7 @@ export default {
         return dpApi.get(url)
       }
 
-      return new Promise((resolve, reject) => resolve(true))
+      return new Promise.resolve(resolve => resolve(true))
```

Comment:
Is this correct syntax? I think Promise.resolve expects only a value, not a method